views/components/certification/Cert.js

```diff
@@ -0,0 +1,58 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components/macro'
+import CertIcon from '~/views/components/certification/CertIcon'
+import Info from '~/views/components/certification/Info'
+import Title from '~/views/components/certification/Title'
+import Item from '~/views/components/certification/Item'
+import { CERT_ACTIVE_ITEM } from '~/consts/certification'
```

Comment:
all the Const should import in one line
{ CERT_..., left...}

Reply:
Updated